<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>#OSSAM Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QMdiArea" name="MdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionQuitter"/>
    <addaction name="actionEnregistrer"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Outils</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAssets_Manager"/>
   <addaction name="actionAnimations"/>
   <addaction name="actionParticles"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionAssets_Manager">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/editor/icons/icons/assets.png</normaloff>:/editor/icons/icons/assets.png</iconset>
   </property>
   <property name="text">
    <string>Assets Manager</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="actionAnimations">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/editor/icons/icons/anims.svg</normaloff>:/editor/icons/icons/anims.svg</iconset>
   </property>
   <property name="text">
    <string>Animations</string>
   </property>
  </action>
  <action name="actionParticles">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/editor/icons/icons/Particle.png</normaloff>:/editor/icons/icons/Particle.png</iconset>
   </property>
   <property name="text">
    <string>Particules</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>openAssetManager()</slot>
 </slots>
</ui>
